packages/semver/project.json

```diff
@@ -10,7 +10,7 @@
       "options": {
         "push": true,
         "noVerify": true,
-        "postTargets": ["publish", "github"],
+        "postTargets": ["build", "publish", "github"],
```

Comment:
Since `runExecutor` DevKit  function is used under the hood on the `postTarget`option, we need to specify the dependant targets of `publish`

For more context, see #771

Reply:
Should we update the [@jscutlery/semver code examples](https://github.com/jscutlery/semver?tab=readme-ov-file#triggering-executors-post-release) to reflect this new behavior of ngx-deploy-npm and raise awareness among the users that adopt both plugins?

Reply:
@dianjuar I just merged your branch and updated the docs! Thanks for the work.